packages/babel-plugin-redux-saga/README.md

```diff
@@ -19,44 +21,43 @@ function* saga2(){
 Result:
 
 ```js
-var _SAGA_LOCATION = require("redux-saga").SAGA_LOCATION
-
 function* saga1() {
-    yield function reduxSagaSource() {
-        var res = foo(1, 2, 3);
-        res[_SAGA_LOCATION] = {
-            fileName: "{{filename}}",
-            lineNumber: 2,
-            code: "foo(1, 2, 3)"
-        };
-        return res;
-    }();
+    yield Object.defineProperty(call(foo, 1, 2, 3), Symbol.for("@@redux-saga/LOCATION"), {
+        value: {
+            fileName: "{{pathToFile}}",
```

Comment:
in readme it would be more clear to use some dummy path here like, `../src/sagas/index.js`

Reply:
ok